reana/reana_benchmark/analyze.py

```diff
@@ -204,6 +205,11 @@ def _build_execution_progress_plot(
 
     ax.set_ylim(ymin=df["workflow_number"].min() - 1)
 
+    if datetime_range:
+        left = datetime.strptime(datetime_range[0], DATETIME_FORMAT)
+        right = datetime.strptime(datetime_range[1], DATETIME_FORMAT)
+        ax.set_xlim(xmin=left, xmax=right)
```

Comment:
Question: just curious why you chose `xmin` and `xmax` over `left` and `right` parameters? According [docs](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.set_xlim.html) it's the same. Have you spotted any difference?

Reply:
It just happened :) I found an example and it was with `xmin/xmax`. I didn't know about the other option. Nice insight! But, `left/right` looks better (+ I even named unconsciously variables).

Reply:
Done